spec/features/saml/sp_initiated_slo_spec.rb

```diff
@@ -83,6 +83,30 @@
     end
   end
 
+  context 'when logged in to a single SP and using new agency based UUIDs' do
+    let(:user) { create(:user, :signed_up) }
+    let(:xmldoc) { SamlResponseDoc.new('feature', 'logout_assertion') }
```

Comment:
This xmldoc variable doesn't seem to be used here, right?

Reply:
Correct. I'll remove it.